src/models/user.model.ts

```diff
@@ -1,9 +1,9 @@
 import { User as ProtoUser } from "@ak-proto-ts/users/v1/user_pb";
 import { User } from "@type/models";
 
-export function convertUserProtoToModel(protoUser?: ProtoUser): User {
+export function convertUserProtoToModel(protoUser: ProtoUser): User {
 	return {
-		email: protoUser?.data?.email || "",
-		name: protoUser?.data?.name || "",
+		email: protoUser.email || "",
+		name: protoUser.displayName || "",
```

Comment:
this is to support non user environment ?

Reply:
This is to support the change in the users model in the latest `itay/users` branch.